src/lua-function-list.json

```diff
@@ -31,21 +31,22 @@
     "clearSpecialExits": "clearSpecialExits(roomID)",
     "clearUserWindow": "clearUserWindow(name)",
     "clearWindow": "clearWindow([windowName])",
+    "closeMapWidget": "closeMapWidget()",
     "closeMudlet": "closeMudlet()",
     "connectExitStub": "connectExitStub(fromID, direction) or connectExitStub(fromID, toID, [direction])",
     "connectToServer": "connectToServer(host, port, [save])",
     "copy": "copy([windowName])",
+    "Create_map_window": "usableId = createRoomID([minimumStartingRoomId])",
     "createBuffer": "createBuffer(name)",
-    "createButton": "createConsole(consoleName, fontSize, charsPerLine, numberOfLines, Xpos, Ypos)",
-    "createGauge": "createGauge(name, width, height, Xpos, Ypos, gaugeText, r, g, b, orientation)",
-    "createLabel": "createLabel(name, Xpos, Ypos, width, height, fillBackground)",
+    "createButton": "createConsole([name of userwindow], consoleName, fontSize, charsPerLine, numberOfLines, Xpos, Ypos)",
```

Comment:
also improperly formatted on wiki

Reply:
for createButton, we don't really properly document it, should we just remove that item from the wiki?